tests/goth_tests/test_run_ssh.py

```diff
@@ -33,16 +33,24 @@ async def test_run_ssh(
     ssh_verify_connection: bool,
 ) -> None:
 
-    websocat_check = pexpect.spawn("/usr/bin/which websocat")
-    exit_code = websocat_check.wait()
-    if exit_code != 0:
-        raise ProcessLookupError("websocat binary not found, please install it or check your PATH.")
+    if ssh_verify_connection:
+        websocat_check = pexpect.spawn("/usr/bin/which websocat")
+        exit_code = websocat_check.wait()
+        if exit_code != 0:
+            raise ProcessLookupError(
+                "websocat binary not found, please install it or check your PATH. "
+                "You may also skip the connection check by omitting `--ssh-verify-connection`."
+            )
 
     configure_logging(log_dir)
 
     # This is the default configuration with 2 wasm/VM providers
     goth_config = load_yaml(goth_config_path, config_overrides)
 
+    # override the requestor node not to use proxy for API calls
```

Comment:
This comment says what the code is doing, but this is pretty obvious either way.
BUT: I don't know why we don't want to use the proxy (or at least, I'm not sure) - and this information would be much more useful here :)